sparkle/src/components/Button.tsx

```diff
@@ -44,10 +44,10 @@ const textClasses = {
 };
 
 const sizeClasses = {
-  xs: "s-gap-x-1 s-px-3 s-py-1.5",
-  sm: "s-gap-x-1 s-px-4 s-py-2",
-  md: "s-gap-x-1.5 s-px-5 s-py-3",
-  lg: "s-gap-x-3 s-px-6 s-py-4",
+  xs: "s-gap-x-1 s-px-3 s-h-7",
```

Comment:
this may have unintended side effects with previous code if there are Button components in front that have classname set using either py or h => the best would be to check for that and confirm the impact on pages is either inexistant or intended (or change the classname if required)

Reply:
Changing py on a button should be punished by death.
I don't think that's ever happening